russstation/code/modules/farts/living/carbon/human/species/corporate.dm

```diff
@@ -10,13 +10,13 @@
 
 /datum/fart/living/carbon/human/species/corporate/hard_fail(mob/living/user)
 	user.visible_message("<span class='danger'>[user]'s ass is going supercritical!</span>")
-	user.Jitter(28)
-	user.Knockdown(28)
+	user.Jitter(2.8 SECONDS)
+	user.Knockdown(2.8 SECONDS)
 	playsound(user, 'sound/effects/huuu.ogg', 75)
 	sleep(2.4 SECONDS)
```

Comment:
sleeps are generally bad practice so I'll try to get around to replacing this soon.